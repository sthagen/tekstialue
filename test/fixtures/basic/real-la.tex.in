%%%%%%%%TEST_COVERAGE
\begin{longtable}[]{@{}
>{\raggedright\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.2233}}
>{\raggedright\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.6200}}
>{\raggedright\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.0667}}
>{\raggedright\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.0900}}@{}}
\toprule()
\begin{minipage}[b]{\linewidth}\raggedright
a
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
b
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
c cc
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
d dd
\end{minipage} \\
\midrule()
\endhead
\href{https://example.com/A-1}{A-1} &
Blurb & and_that &
Also_this \\
\href{https://example.com/A-2}{A-2} &
Blurb too & and_these &
Also_those \\
\bottomrule()
\end{longtable}

2 issues

\hypertarget{aaa-aaaaaaa}{%
\subsubsection{Aaaa Aaaadasda}\label{aaa-aaaaaaa}}
