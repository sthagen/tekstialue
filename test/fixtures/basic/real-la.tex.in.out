
%%%%%%%%TEST_COVERAGE
\begin{small}
\begin{longtable}[]{|
>{\raggedright\arraybackslash}p{(\columnwidth - 12\tabcolsep) * \real{0.1500}}|
>{\raggedright\arraybackslash}p{(\columnwidth - 12\tabcolsep) * \real{0.5500}}|
>{\raggedright\arraybackslash}p{(\columnwidth - 12\tabcolsep) * \real{0.1500}}|
>{\raggedright\arraybackslash}p{(\columnwidth - 12\tabcolsep) * \real{0.2000}}|}
\hline
\begin{minipage}[b]{\linewidth}\raggedright
\ \mbox{\textbf{a}}
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
\mbox{\textbf{b}}
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
\mbox{\textbf{c}} \mbox{\textbf{cc}}
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
\mbox{\textbf{d}} \mbox{\textbf{dd}}
\end{minipage} \\
\hline
\endfirsthead
\multicolumn{4}{@{}l}{\small \ldots continued}\\\hline
\hline
\begin{minipage}[b]{\linewidth}\raggedright
\ \mbox{\textbf{a}}
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
\mbox{\textbf{b}}
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
\mbox{\textbf{c}} \mbox{\textbf{cc}}
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
\mbox{\textbf{d}} \mbox{\textbf{dd}}
\end{minipage} \\
\endhead
\hline
\href{https://example.com/A-1}{A-1} &
Blurb & and_that &
Also_this \\
\hline
\href{https://example.com/A-2}{A-2} &
Blurb too & and_these &
Also_those \\
\hline
\end{longtable}
\end{small}
\vspace*{-2em}
\begin{footnotesize}
2 issues
\end{footnotesize}

\hypertarget{aaa-aaaaaaa}{%
\subsubsection{Aaaa Aaaadasda}\label{aaa-aaaaaaa}}
